lib/transaction.ex

```diff
@@ -31,6 +31,67 @@ defmodule Elixium.Transaction do
     %{outputs: outputs}
   end
 
+  @doc """
+  Creates a correct tx id
+  """
+  @spec create_tx_id(List, String.t()) :: String.t()
```

Comment:
Why merge timestamp with the tx hash? The tx id should just be the result of calculate_hash(tx)

Reply:
Fixed & Removed